libwasmvm/Cargo.toml

```diff
@@ -13,25 +13,25 @@ exclude = [".circleci/*", ".gitignore"]
 [lib]
 crate-type = ["cdylib"]
 
-# the example is to allow us to compile a static lib with the same codebase as we compile the
+# the example is to allow us to compile a muslc static lib with the same codebase as we compile the
 # normal dynamic libs (best workaround I could find to override crate-type on the command line)
 [[example]]
 name = "muslc"
 path = "src/lib.rs"
 crate-type = ["staticlib"]
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-[badges]
-maintenance = { status = "actively-developed" }
+# # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+# [badges]
+# maintenance = { status = "actively-developed" }
 
 [features]
 default = []
 # This feature requires Rust nightly because it depends on the unstable backtrace feature.
 backtraces = []
 
 [dependencies]
-cosmwasm-std = { git = "https://github.com/line/cosmwasm", tag = "v1.0.0-0.6.0", features = ["iterator","staking","stargate"] }
-cosmwasm-vm = {  git = "https://github.com/line/cosmwasm", tag = "v1.0.0-0.6.0", features = ["iterator","staking","stargate"] }
+cosmwasm-std = { git = "https://github.com/line/cosmwasm", rev = "refs/pull/256/head", features = ["iterator","staking","stargate"]}
+cosmwasm-vm = { git = "https://github.com/line/cosmwasm", rev = "refs/pull/256/head", features = ["iterator","staking","stargate"]}
```

Comment:
I'll change this git path after merging https://github.com/line/cosmwasm/pull/256 PR.